src/System.CommandLine/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("System.CommandLine.Rendering")]
```

Comment:
Would love to try and restructure to remove this.

Reply:
Me too. I'm trying to figure that out now, since our build tools make it awkward to make this work both in CI (where there's a strong name) and locally (where there is not).

`ITerminal` seems like it potentially belongs in `System.CommandLine` but it still needs trimming down so that it doesn't drag other things along. Thoughts?